host/tests/L3_TestCases/dsDisplay/dsDisplayHelperClass.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, testName:str, qcId:str, log:logModule=None ):
 
```

Comment:
Need to update test spec document for L3

Reply:
updated the test specification document for L3